README.md

```diff
@@ -106,8 +108,7 @@ bindkey '^ ' autosuggest-accept
 
 ## Troubleshooting
 
-If you have a problem, please search through [the list of issues on GitHub](https://github.com/zsh-users/zsh-autosuggestions/issues) to see if someone else has already reported it.
-
```

Comment:
This whitespace is actually intentional, can update the commit to retain it?

Reply:
Actually, you know what, the whitespace is already inconsistent in this file. Gonna go ahead and merge, and I'll clean up some time in the future.